sql/catalyst/src/main/scala/org/apache/spark/sql/types/StructField.scala

```diff
@@ -74,4 +74,11 @@ case class StructField(
   def getComment(): Option[String] = {
     if (metadata.contains("comment")) Option(metadata.getString("comment")) else None
   }
+
+  /**
+   * Return the full description of the StructField.
+   */
+  def getDesc(): String = {
```

Comment:
Yes because we have to print the full description of the StructField, but `StructField.toString` don't output `metadata` field.